### DISCUSSION TITLE
Polyphenyl control seems to have a small display bug. Do we need to fix it?

### DISCUSSION THREAD

**Discussion by erdengk** [polypheny/Polypheny-DB#374] [Q&A] [answered]
@vogti 
I'm Kevin
I found this problem when I ran polyphenyl control.
Polyphene control seems to have a small bug. I use the Chrome browser. After downloading the file, the footer bar at the bottom will cover the OK button and can't be used normally. I think this is a place that needs to be modified.
As shown in the figure below:

![WX20211214-214526](https://user-images.githubusercontent.com/37730787/146151742-7fcb9a9c-0c50-4ef1-974b-36d2e82359d8.png)

If it needs to be fixed, I have tried to fix this bug, which can ensure that the footer bar will not block the OK button.

<img width="1199" alt="WX20211215-162610@2x" src="https://user-images.githubusercontent.com/37730787/146151477-eacbac74-21ac-4d5d-8a68-931b208a0c8f.png">

Do I need to submit PR？

**Comment by vogti** [comment, marked as answer]
Hi Kevin,
Thanks for pointing out this issue! It would be great if you can create a PR with a fix.

**Reply by erdengk** [reply]
@vogti, please review the PR mentioned below. Thanks.
[Polypheny-Control/pull/15](https://github.com/polypheny/Polypheny-Control/pull/15)
